examples/siem-connector/readme.md

```diff
@@ -1,54 +0,0 @@
-# SIEM Connector Example
```

Comment:
Accidental delete? We still have the readme for this in lepton

Reply:
It looks like this PR deletes both, is the plan to have the sync bring the correct one across when the JIRA issue is resolved?